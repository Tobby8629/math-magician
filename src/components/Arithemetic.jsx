import React from 'react';
import Calculator from './Calculator';

const Arithemetic = () => (
  <div className="arit">
    <Calculator />
  </div>
);
export default Arithemetic;
