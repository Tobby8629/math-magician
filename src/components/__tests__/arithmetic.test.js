import renderer from 'react-test-renderer';
import Arithemetic from '../Arithemetic';

test('Arithmetic component match snapshot', () => {
  const tree = renderer.create(<Arithemetic />).toJSON;
  expect(tree).toMatchSnapshot();
});
