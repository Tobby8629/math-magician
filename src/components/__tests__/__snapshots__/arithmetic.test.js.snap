// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arithmetic component match snapshot 1`] = `[Function]`;
